spec/lib/embed/request_spec.rb

```diff
@@ -45,17 +37,13 @@
 
   describe 'as_url_params' do
     let(:url_params) do
-      Embed::Request.new(url: purl, hide_title: 'true', hide_metadata: 'true').as_url_params
+      Embed::Request.new(url: purl, hide_title: 'true').as_url_params
     end
 
     it 'is a hash of parameters to be turned into url optinos' do
       expect(url_params).to be_a Hash
       expect(url_params[:hide_title]).to eq('true')
     end
-
-    it 'does not include parameters that should not be passed along in a url (such as hide_metadata)' do
```

Comment:
👍 I'll add a test back in.  This is whitelisting so testing that other things aren't making their way in is a good idea.  Good catch.